### ISSUE TITLE
Firefox 48.0 warning (no error)

### ISSUE THREAD

**Issue by ghost** [jhdrn/myra#5]
Using document.createAttribute () it is deprecated. Use in place element.setAttribute ().

view-source:http://localhost:8080/bundle.js:8464:20

```
        var attr = document.createAttribute(attributeName);
```

**Comment by jhdrn** [comment]
Yep,  it's actually fixed, but not published to NPM. Will publish a new package soon.

**Comment by jhdrn** [comment]
I have now published version 1.0.0-alpha.20160820 to NPM. It should fix those warnings.
